### PR TITLE
feat(BannerUpgrade): add tests

### PR DESCRIPTION
**Pull request by lucianaDilizio**
### Banner Upgade
- Add tests to `BannerUpgrade.js` component.

### CODE REVIEW

File: src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    expect(container.querySelector('.dp-calc-message')).toBeInTheDocument();
+  });
+
+  describe.each([
+    { planType: types.MONTHLY_DELIVERIES },
+    { planType: types.PREPAID },
+    { planType: types.SUSCRIBERS },
+  ])('Content validation', ({ planType }) => {
+    it(`should show correct text in banner for ${planType} plan type`, () => {
+      // Act
```
Comment (jcamposmk) [inline src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js:103]: Arrange

Comment (jcamposmk) [inline src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js:103]: // Arrange

File: src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        type: planType,
+        featureSet: featuresSets.STANDARD,
+      };
+
+      const currentPlanList = [];
+      for (let i = 1; i <= 10; i++) {
+        currentPlanList.push({ ...currentPlan, id: i });
+      }
+
+      const { getByText } = render(
```
Comment (jcamposmk) [inline src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js:115]: // Act

File: src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        <BannerUpgrade currentPlan={currentPlan} currentPlanList={currentPlanList} />
+      </DependenciesContainer>,
+    );
+
+    // Assert
+    expect(container.querySelector('.dp-calc-message')).not.toBeInTheDocument();
+  });
+
+  it('should show banner when user is positioned on the maximum plan for the current feature set', () => {
+    // Act
```
Comment (jcamposmk) [inline src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js:52]: Arrange

File: src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+      type: types.MONTHLY_DELIVERIES,
+      featureSet: featuresSets.STANDARD,
+    };
+
+    const currentPlanList = [];
+    for (let i = 1; i <= 10; i++) {
+      currentPlanList.push({ ...currentPlan, id: i });
+    }
+
+    const { container } = render(
```
Comment (jcamposmk) [inline src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js:64]: Act

File: src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js (hunk 3 above)
Comment (lucianaDilizio) [reply src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js:52]: I get it, copy/paste mistakes 😅

Comment (lucianaDilizio) [reply src/components/ChangePlan/PlanCalculator/BannerUpgrade/BannerUpgrade.test.js:52]: Thanks!

**Comment by lucianaDilizio** [comment]
Test this please 🙏

**Pull request merged**